VARIABLES.md

```diff
@@ -220,6 +220,14 @@ Default:  /usr/local/bin/confluent
 
 ***
 
+### default_internal_replication_factor
+
+Recommended replication factor. Usually 3. If running in 2DC and look for balanced replicad, consider increasing to 4. If number of brokers is less than this property, then minimun value is used.
```

Comment:
```suggestion
Recommended replication factor, defaults to 3. When splitting your cluster across 2 DCs with 4 or more Brokers, this should be increased to 4 to balance topic replicas.```

Reply:
Make sure these changes get reflected in both the VARIABLES.md and the place annotation originates